src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -156,6 +180,7 @@
      * @param w  The world the colony exists in.
      * @param c  The center of the colony (location of Town Hall).
      */
+    @SuppressWarnings("squid:S2637")
```

Comment:
Don't suppress this, use temporary variables marked @NotNull to circumvent these dumb forge defaults